### PR TITLE
[IMP] developer: document formatters/parsers registries

### PR DESCRIPTION
**Pull request by ged-odoo**
and add entry for effects registry as well

### CODE REVIEW

File: content/developer/reference/frontend/registries.rst [hunk 1]
```diff
@@ -106,15 +106,50 @@ Reference List
...
+has the following API:
+
+.. js:function:: format(value[, options])
+
+    :param value: a value of a specific type, or `false` if no value is given
+    :type value: T | false
+    :param Object options: various options
+    :returns: string
+
+    Format a value and returns a string
```
Comment (aab-odoo) [inline content/developer/reference/frontend/registries.rst:148]: Format and return
or
Formats and returns
but not a mix :-)

File: content/developer/reference/frontend/registries.rst [hunk 2]
```diff
@@ -143,6 +178,28 @@ this:
...
+
+.. js:function:: parse(value[, options])
+    :noindex:
+
+    :param value: a string representing a value
+    :type value: string
+    :param Object options: various options (parser specific)
+    :returns: T a valid value 
+
+    Parse a string and returns a value. If the string does not represent a valid
```
Comment (aab-odoo) [inline content/developer/reference/frontend/registries.rst:197]: same

File: content/developer/reference/frontend/registries.rst [hunk 3]
```diff
@@ -106,15 +106,50 @@ Reference List
...
+----------------
+
+The `effects` registry contains the implementations of all available effects.
+See the section on the :ref:`effect service <frontend/services/effect_registry>`
+for more details.
+
+.. _frontend/registries/formatters:
+
+Formatters registry
+-------------------------
```
Comment (SimonGenin) [inline content/developer/reference/frontend/registries.rst:136]: too long title line

File: content/developer/reference/frontend/registries.rst [hunk 4]
```diff
@@ -106,15 +106,50 @@ Reference List
...
    * - :ref:`services <frontend/registries/services>`
      - all services that should be activated
    * - :ref:`systray <frontend/registries/systray>`
      - components displayed in the systray zone in the navbar
    * - :ref:`user_menuitems <frontend/registries/usermenu>`
      - menu items displayed in the user menu (top right of navbar)
 
+.. _frontend/registries/effects:
+
+Effects registry
```
Comment (SimonGenin) [inline content/developer/reference/frontend/registries.rst:126]: not sure effects should be plural

File: content/developer/reference/frontend/registries.rst [hunk 5]
```diff
@@ -143,6 +178,28 @@ this:
      Component: LoadingIndicator,
    });
 
+.. _frontend/registries/parsers:
+
+Parsers registry
```
Comment (SimonGenin) [inline content/developer/reference/frontend/registries.rst:183]: not sure parsers should be plural

**Review by Feyensv — APPROVED** [review]
@robodoo delegate+

File: content/developer/reference/frontend/registries.rst (hunk 4 above)
Comment (AntoineVDV) [reply content/developer/reference/frontend/registries.rst:126]: If that's the name of the registry it's good as it is.

**Comment by ged-odoo** [comment]
@robodoo r+

**Pull request closed without merging**